include/mbedtls/mbedtls_config.h

```diff
@@ -3136,8 +3140,8 @@
  * Requires: MBEDTLS_ASN1_PARSE_C, MBEDTLS_BIGNUM_C, MBEDTLS_OID_C, MBEDTLS_PK_PARSE_C,
  *           (MBEDTLS_MD_C or MBEDTLS_USE_PSA_CRYPTO)
  *
- * \warning If building without MBEDTLS_MD_C, you must call psa_crypto_init()
- * before doing any X.509 operation.
+ * \warning If building with MBEDTLS_USE_PSA_CRYPTO, you must call
```

Comment:
Is this the case here and in other places? Wasn't `MBEDTLS_MD_C` taking precedence and is used even though `MBEDTLS_USE_PSA_CRYPTO` is defined?

Reply:
Right, that was the case for other modules.

Reply:
For modules that are affected by `MBEDTLS_USE_PSA_CRYPTO`, that's what decides whether PSA is used or not, for the others indeed MD has precedence and is used when available.

Note that for this module, building without `MD_C` means you are building with `USE_PSA_CRYPTO`, so the new condition encompasses the old one.